lib/src/argument_list_visitor.dart

```diff
@@ -224,6 +146,83 @@ class ArgumentListVisitor {
     if (_isSingle) _visitor.builder.endSpan();
   }
 
+  /// Look for a single contiguous range of block function [arguments] that
+  /// should receive special formatting.
+  ///
+  /// Returns a list of (start, end] indexes if found, otherwise returns `null`.
+  static List<int>? _contiguousFunctions(List<Expression> arguments) {
+    int? functionsStart;
```

Comment:
This code is essentially unchanged from line 66 on. The main difference is that the old code used to use `arguments[0] is NamedExpression` as a clever check to see if all arguments are named. That doesn't work with named args anywhere so now it iterates the list and checks them all.